package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@salesforce/lwc-dev-mobile-core",
   "description": "Core module supporting Salesforce CLI mobile extension plug-ins",
-  "version": "1.0.0",
+  "version": "1.0.0-beta12",
```

Comment:
should it be `1.1.0-beta1` instead? 🤔

Reply:
Yes—or whatever x.y.z-beta1 we choose where x or y or z needs to increment past where we are today.

Reply:
I'm also not sure that we should go the linear route from what `main` is versioning, either. If we need to make another release (in any capacity) of the -core package before we promote this, we're going to get into confusing territory, version-wise.

I'd propose the following:

- Version this as `1.0.0-lwr1`. The suffix doesn't have to be strict to "beta".
- When we publish, publish this with the "lwr" tag, rather than the "beta" tag.

Given that this is a feature branch with an indeterminate longevity—rather than a mainline continuance—I would prefer we keep the versioning separate, between them.

Reply:
oh I didn't know `npm` allows us to put anything we want after the version. I thought `beta` was a special case. In that case `1.0.0-lwr` is the best option.